src/lib/viem-client.ts

```diff
@@ -2,7 +2,7 @@ import { getConfig } from 'config'
 import { createPublicClient, http } from 'viem'
 import { mainnet } from 'viem/chains'
 
-export const viemClient = createPublicClient({
+export const viemEthMainnetClient = createPublicClient({
```

Comment:
renamed, as this was mega confusing when looking at the deleted Web3ModalProvider which was using viemClient which seemed generalized but absolutely was not